src/platform/packages/shared/kbn-unified-histogram/components/chart/chart_section_template.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import type { SerializedStyles } from '@emotion/serialize';
+import { i18n } from '@kbn/i18n';
+import { IconButtonGroup, type IconButtonGroupProps } from '@kbn/shared-ux-button-toolbar';
+import React from 'react';
+
+interface ChartSectionTemplateProps {
+  id: string;
+  toolbarCss?: SerializedStyles;
+  toolbar?: {
+    rightSide?: React.ReactNode;
+    leftSide?: IconButtonGroupProps['buttons'];
+  };
+  children: React.ReactNode;
+}
+
+export const ChartSectionTemplate = ({
+  id,
+  toolbarCss,
+  toolbar,
+  children,
+}: ChartSectionTemplateProps) => {
+  const { leftSide = [], rightSide = [] } = toolbar ?? {};
+
+  return (
+    <EuiFlexGroup
+      id={id}
+      className="unifiedHistogram__chart"
+      direction="column"
+      alignItems="stretch"
+      gutterSize="none"
+      responsive={false}
+    >
+      <EuiFlexItem grow={false} css={toolbarCss}>
+        <EuiFlexGroup
+          direction="row"
+          gutterSize="s"
+          responsive={false}
+          alignItems="center"
+          justifyContent="spaceBetween"
+        >
+          <EuiFlexItem grow={false} css={{ minWidth: 0 }}>
+            <EuiFlexGroup direction="row" gutterSize="s" responsive={false} alignItems="center">
+              {React.Children.toArray(rightSide).map((child, i) => (
```

Comment:
```suggestion
              {rightSide.map((child, i) => (
```

Can it be simplified?